### PR TITLE
Feature dismiss error message

### LINKED ISSUE #555: Add acknowledge button on error messages on front page
**Describe the improvement you would like to see**
Add acknowledge button on error message. Error message should disappear when the acknowledge button is pressed. 

Current solution:
image

**How will this change existing functionality?**
Front page will get a cleaner look when error messages can be acknowledged and removed.

**How will this improvement affect the current Threat Model?**

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #555

### CODE REVIEW

File: frontend/src/components/Pages/FrontPage/MissionOverview/FailedMissionAlertView.tsx [hunk 1]
```diff
@@ -47,17 +59,39 @@ export function FailedMissionAlertView({ refreshInterval }: RefreshProps) {
 
     const apiCaller = useApi()
     const [missionsToDisplay, setMissionsToDisplay] = useState<Mission[]>([])
+    const [dismissedMissions, setDismissedMissions] = useState<Mission[]>([])
+    const [recentFailedMissions, setRecentFailedMissions] = useState<Mission[]>([])
```
Comment (Eddasol) [inline frontend/src/components/Pages/FrontPage/MissionOverview/FailedMissionAlertView.tsx:63]: Would it be enough with just failedMissionsToDisplay?

**Comment by GodVenn** [comment]
Will change dismissed missions to be added to session storage for "persistent data per user"

**Review by Eddasol — COMMENTED** [review]
Suggest changing such that all failed missions since last dismiss is shown and not just last 10 minutes

**Comment by GodVenn** [comment]
> Suggest changing such that all failed missions since last dismiss is shown and not just last 10 minutes

I would argue that's what we have the "Past mission" section for and that this alert is meant as a "right now"-alert?

**Review by Eddasol — APPROVED** [review]
LGTM

**Pull request merged**